packages/core/src/batch/ObjectRenderer.js

```diff
@@ -1,13 +1,13 @@
-import WebGLSystem from '../systems/WebGLSystem';
+import System from '../System';
 
 /**
  * Base for a common object renderer that can be used as a system renderer plugin.
  *
  * @class
- * @extends PIXI.WebGLSystem
+ * @extends PIXI.System
```

Comment:
namespace missing

Reply:
`PIXI.System` is the main class and `PIXI.systems.*`  are where the actual system classes will live like TextureSystem, FilterSystem, etc.